### PR TITLE
update addon enquque ns and addon  update message

### PR DESCRIPTION
**Pull request by skeeey**
Signed-off-by: liuwei <liuweixa@redhat.com>

### CODE REVIEW

File: test/integration/addon_lease_test.go [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+		assertAddOn()
+		gomega.Eventually(func() bool {
+			addOn, err := addOnClient.AddonV1alpha1().ManagedClusterAddOns(managedClusterName).Get(context.TODO(), addOnName, metav1.GetOptions{})
+			if err != nil {
+				return false
+			}
+			return meta.IsStatusConditionTrue(addOn.Status.Conditions, "Available")
+		}, eventuallyTimeout, eventuallyInterval).Should(gomega.BeTrue())
+
+		// do not update addon lease, wait once resync, the addon status should be unavailable
```
Comment (qiujian16) [inline test/integration/addon_lease_test.go]: wait resync once

**Review by qiujian16 — COMMENTED** [review]
/approve

File: test/integration/addon_lease_test.go (hunk 1 above)
Comment (skeeey) [reply test/integration/addon_lease_test.go]: corrected

**Pull request merged**